portal/src/main/webapp/js/src/oncokb/OncoKBConnector.js

```diff
@@ -157,6 +157,7 @@ var OncoKB = (function(_, $) {
     function Instance(id) {
         this.initialized = false;
         this.dataReady = false;
+        this.civicDataReady = false;
```

Comment:
I also would prefer something like `CivicConnector.js` instead. But if there are refactoring plans anyway, perhaps it is OK for now.